examples/notebooks/Pivoting Data from Wide to Long.ipynb

```diff
@@ -708,13 +936,176 @@
+       "\n",
+       "   Sanyo | Radio | Max-quant | value  \n",
+       "0                                  9  \n",
+       "1                                  9  \n",
+       "2                                 10  "
+      ],
+      "text/html": "<div>\n<style scoped>\n    .dataframe tbody tr th:only-of-type {\n        vertical-align: middle;\n    }\n\n    .dataframe tbody tr th {\n        vertical-align: top;\n    }\n\n    .dataframe thead th {\n        text-align: right;\n    }\n</style>\n<table border=\"1\" class=\"dataframe\">\n  <thead>\n    <tr style=\"text-align: right;\">\n      <th></th>\n      <th>Sony | TV | Model | value</th>\n      <th>Sony | TV | Quantity | value</th>\n      <th>Sony | TV | Max-quant | value</th>\n      <th>Panasonic | TV | Model | value</th>\n      <th>Panasonic | TV | Quantity | value</th>\n      <th>Panasonic | TV | Max-quant | value</th>\n      <th>Sanyo | Radio | Model | value</th>\n      <th>Sanyo | Radio | Quantity | value</th>\n      <th>Sanyo | Radio | Max-quant | value</th>\n    </tr>\n  </thead>\n  <tbody>\n    <tr>\n      <th>0</th>\n      <td>A222</td>\n      <td>5</td>\n      <td>10</td>\n      <td>T232</td>\n      <td>1</td>\n      <td>10</td>\n      <td>S111</td>\n      <td>4</td>\n      <td>9</td>\n    </tr>\n    <tr>\n      <th>1</th>\n      <td>A234</td>\n      <td>5</td>\n      <td>9</td>\n      <td>S3424</td>\n      <td>5</td>\n      <td>12</td>\n      <td>S1s1</td>\n      <td>2</td>\n      <td>9</td>\n    </tr>\n    <tr>\n      <th>2</th>\n      <td>A4345</td>\n      <td>4</td>\n      <td>9</td>\n      <td>X3421</td>\n      <td>1</td>\n      <td>11</td>\n      <td>S1s2</td>\n      <td>4</td>\n      <td>10</td>\n    </tr>\n  </tbody>\n</table>\n</div>"
      },
      "metadata": {},
      "execution_count": 23
     }
    ],
+   "source": [
+    "df = pd.DataFrame(\n",
+    "    {\n",
+    "        \"Sony | TV | Model | value\": {0: \"A222\", 1: \"A234\", 2: \"A4345\"},\n",
+    "        \"Sony | TV | Quantity | value\": {0: 5, 1: 5, 2: 4},\n",
+    "        \"Sony | TV | Max-quant | value\": {0: 10, 1: 9, 2: 9},\n",
+    "        \"Panasonic | TV | Model | value\": {0: \"T232\", 1: \"S3424\", 2: \"X3421\"},\n",
+    "        \"Panasonic | TV | Quantity | value\": {0: 1, 1: 5, 2: 1},\n",
+    "        \"Panasonic | TV | Max-quant | value\": {0: 10, 1: 12, 2: 11},\n",
+    "        \"Sanyo | Radio | Model | value\": {0: \"S111\", 1: \"S1s1\", 2: \"S1s2\"},\n",
+    "        \"Sanyo | Radio | Quantity | value\": {0: 4, 1: 2, 2: 4},\n",
+    "        \"Sanyo | Radio | Max-quant | value\": {0: 9, 1: 9, 2: 10},\n",
+    "    }\n",
+    ")\n",
+    "\n",
+    "df"
+   ]
+  },
+  {
+   "source": [
+    "The goal is to reshaped the data into long format, with separate columns for `Manufacturer`(Sony,...), `Device`(TV, Radio), `Model`(S3424, ...), maximum quantity and quantity. \n",
+    "\n",
+    "A good solution was proferred, also on [Stack Overflow](https://stackoverflow.com/a/64107688/7175713) :"
```

Comment:
proffered?

Reply:
I will change this to something else, which should be clearer.